src/submission/validation-clinical/utils.ts

```diff
@@ -112,14 +112,11 @@ const getSubmissionUpdates = (clinicalObject: any, record: SubmittedClinicalReco
     for (const fieldName in record) {
       // skip the index field
       if (fieldName == 'index') continue;
-      // new field added not in the old object
-      if (!clinicalObject[fieldName]) {
```

Comment:
fields values that were `null` in clinicalInfo were being returned as updates because this evaluated as true, which would've been fine if the record value wasn't also `null`

Reply:
so my question is why would `record[fieldName]` be undefined/null ?

Reply:
I believe if you pass a header with no values in the column, that record[fieldName] will be undefined because the processed records from schema processing returns them like that. Then when undefined is stored in mongo it becomes null.